client/src/components/JurisdictionAdmin/JurisdictionAdminView.tsx

```diff
@@ -77,6 +78,14 @@ const JurisdictionAdminView: React.FC = () => {
     j => j.id === jurisdictionId
   )!
 
+  const isManifestUploaded = isFileProcessed(ballotManifest)
+  const isBatchTalliesUploaded =
+    !isBatchComparison || isFileProcessed(batchTallies!)
+  const isCvrsUploaded =
+    !(isBallotComparison || isHybrid) || isFileProcessed(cvrs!)
+  const allFilesUploaded =
```

Comment:
Another nit that you're free to ignore: For consistency, could name this `areAllFilesUploaded`

Reply:
Ha! I thought about that after I put the PR up and decided to leave it. I'll fix